blocks/init/src/Blocks/custom/tab/tab-block.js

```diff
@@ -1,31 +1,49 @@
 import React from 'react';
 import { getUnique, overrideInnerBlockSimpleWrapperAttributes } from '@eightshift/frontend-libs/scripts/editor';
 import { checkAttr, getAttrKey } from '@eightshift/frontend-libs/scripts/helpers';
-import { InspectorControls } from '@wordpress/block-editor';
 import { TabEditor } from './components/tab-editor';
-import { TabOptions } from './components/tab-options';
 import { useSelect } from '@wordpress/data';
 import manifest from './manifest.json';
 
 export const Tab = (props) => {
-	const {attributes, clientId, setAttributes} = props;
+	const {
+		attributes,
+		clientId,
+		setAttributes
+	} = props;
+
+	const parentClientId = wp.data.select('core/block-editor').getBlockParentsByBlockName(clientId, ['eightshift-boilerplate/tabs'])[0];
+
+	let activeTab;
+	let setActiveTab;
+
+	if (typeof(attributes.setActiveTab) !== 'undefined') {
+		activeTab = attributes.activeTab;
+		setActiveTab = attributes.setActiveTab;
+	}
+	else {
+		const parentBlock = wp.data.select('core/block-editor').getBlockAttributes(parentClientId);
+		activeTab = parentBlock.activeTab;
+		setActiveTab = parentBlock.setActiveTab;
+	}
+
 	useSelect((select) => {
 		overrideInnerBlockSimpleWrapperAttributes(select, clientId);
 	});
+
 	let tabId = checkAttr('tabId', attributes, manifest, true);
-	if(tabId === "newTab") {
+
+	if (tabId === undefined) {
```

Comment:
```suggestion
	if (typeof(tabId) === 'undefined') {
```

I think we usually check against the string.